src/main/java/org/motechproject/mots/scheduler/ExampleJob.java

```diff
@@ -0,0 +1,59 @@
+package org.motechproject.mots.scheduler;
+
+import java.time.Instant;
+import java.util.Date;
+import lombok.extern.slf4j.Slf4j;
+import org.quartz.JobExecutionContext;
+import org.quartz.Trigger;
+import org.quartz.TriggerBuilder;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+@Slf4j
```

Comment:
in mots we're using `private static final Logger LOGGER = LoggerFactory.getLogger(ExampleJob.class);` for logging